### PR TITLE
Option to skip search engine indexing inside eZContentOperationCollection::updateSection

### PR DESCRIPTION
**Pull request by mateuszbieniek**
This PR is a follow up for https://github.com/ezsystems/ezpublish-legacy/pull/1378

### CODE REVIEW

File: kernel/content/ezcontentoperationcollection.php [hunk 1]
```diff
@@ -1152,12 +1152,13 @@ static public function swapNode( $nodeID, $selectedNodeID, $nodeIdList = array()
      *
      * @param int $nodeID
      * @param int $selectedSectionID
+     * @param bool $updateSearchIndexes
      *
-     * @return array An array with operation status, always true
+     * @return void An array with operation status, always true
```
Comment (glye) [inline kernel/content/ezcontentoperationcollection.php]: Uh...? It returns nothing, specifically an array? :)

**Comment by mateuszbieniek** [comment]
Good catch @glye :grinning:

**Comment by andrerom** [comment]
Thanks @mateuszbieniek, I'll cherry pick to 5.4 ;)

**Comment by andrerom** [comment]
@pkamps hope this solves your issue with the first patch

**Comment by pkamps** [comment]
Thanks for addressing the issue.

**Pull request merged**